### PR TITLE
Adjust max-delay for meanVtx calib

### PR DESCRIPTION
**Pull request by chiarazampolli**
@shahor02

### CODE REVIEW

**Comment by chiarazampolli** [comment]
@shahor02 , if you agree with this change, can you merge it?

**Comment by chiarazampolli** [comment]
@shahor02 , as we discussed.

File: prodtests/full-system-test/aggregator-workflow.sh [hunk 1]
```diff
@@ -148,7 +148,10 @@ fi
 if [[ $AGGREGATOR_TASKS == BARREL_TF ]] || [[ $AGGREGATOR_TASKS == ALL ]]; then
   # PrimVertex
   if [[ $CALIB_PRIMVTX_MEANVTX == 1 ]]; then
-    add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=55000"
+    if [[ -z $TFPERSLOTS_MEANVTX ]]; then TFPERSLOTS_MEANVTX=55000; fi
+    DELAYINTFS_MEANVTX="10"
+    MAXDELAY_INSLOTFRACTION_MEANVTX=`awk "BEGIN {print $DELAYINTFS_MEANVTX/$TFPERSLOTS_MEANVTX}"`
+    add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX --max-delay $MAXDELAY_INSLOTFRACTION_MEANVTX"
```
Comment (shahor02) [inline prodtests/full-system-test/aggregator-workflow.sh]: After checking mean-vertex code I see that there is no --max-delay but only `MeanVertexParams::maxTFdelay`.
Should not it be 
```
add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX;MeanVertexCalib.maxTFdelay=$MAXDELAY_INSLOTFRACTION_MEANVTX"
```
?

Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh]: Yes, fixed

**Comment by chiarazampolli** [comment]
Hi, 
But wait: for meanvertex this is a uint32_t. Let me check better. I will put this PR in WIP.

**Comment by shahor02** [comment]
Perhaps you can interpret the MeanVertexCalib.maxTFdelay as number of TFs (as the name suggests) and convert it to float in 
https://github.com/AliceO2Group/AliceO2/blob/f722a051134255d91fcdf46499c49714e09bd5a1/Detectors/Calibration/workflow/src/MeanVertexCalibratorSpec.cxx#L47
as 
```
mCalibrator->setMaxSlotsDelay(float(delay)/slotL);
```
This will be a cleaner solution than doing floating point arithmetic in the script

**Comment by chiarazampolli** [comment]
Yes, I will do that. I was debugging the EPN grid job issue with the others before.

**Comment by chiarazampolli** [comment]
Done, but I did not compile (ongoing). Let's wait for the tests.

**Comment by chiarazampolli** [comment]
Hello @shahor02 , 
The CI is green, do you think it can be merged now?
Chiara

**Comment by chiarazampolli** [comment]
Why is it in error now? @TimoWilken , @ktf , @shahor02

**Review by chiarazampolli — COMMENTED** [review]
For me the conversion was working.
```
root [0] const o2::calibration::MeanVertexParams* params = &o2::calibration::MeanVertexParams::Instance();
root [1] params->getM
variable "params->getM" not defined.
root [1] params->max
variable "params->max" not defined.
root [1] params->maxTFdelay
(const unsigned int) 3
root [2] int slotL = 55000
(int) 55000
root [3] float n = (float)params->maxTFdelay / slotL
(float) 5.45455e-05f
root [4] float n = float(params->maxTFdelay) / slotL
(float) 5.45455e-05f
root [5] 
```
The result is the same with both ways.

**Review by shahor02 — APPROVED** [review]
I should not have called this a typo, c-style cast indeed has higher precedence than arithmetic operations, but the function-style case is more explicit. 
Anyway, it can be merged,

**Pull request merged**